ceph.spec.in

```diff
@@ -292,6 +282,12 @@ BuildRequires:	lz4-devel >= 1.7
 %endif
 # distro-conditional make check dependencies
 %if 0%{with make_check}
+%if 0%{?fedora} || 0%{?rhel}
+%if 0%{?fedora} || 0%{?rhel} < 8
```

Comment:
can we move following three dependencies into `if 0%{?fedora} || 0%{?rhel} == 7` branch? as only RHEL/CentOS 7 are supported "if 0%{?fedora} || 0%{?rhel} < 8".

Reply:
Happy to do so, done.

Reply:
@tchaikov I moved them into `if 0%{?fedora} || 0%{?rhel} == 7` as you wrote, but I'm wondering whether you actually meant to move them into the `if 0%{?rhel} == 7` block - i.e., to exclude Fedora?

Reply:
@smithfarm no, this is exactly what i want. no, i don't want to exclude fedora. otherwise this PR changes the existing behavior.